### PR TITLE
Update Readmes in packages

### PR DESCRIPTION
**Pull request by magicmatatjahu**
**Description**

Changes proposed in this pull request:
- Update Readmes in packages: add info how import default styles in render engines.

### CODE REVIEW

File: packages/async-api-render-engine/README.md [hunk 1]
```diff
@@ -18,10 +18,6 @@ The AsyncAPI render engine is based on the [AsyncAPI React](https://github.com/a
   yarn add @kyma-project/dc-async-api-render-engine
   ```
 
-## Prerequisites
```
Comment (aerfio) [inline packages/async-api-render-engine/README.md]: maybe instead of deleting this section add asyncapi-react as prerequisite?

Comment (derberg) [reply packages/async-api-render-engine/README.md]: but I do not need `asyncapi-react`, like I should not need `odata-react`

Comment (magicmatatjahu) [reply packages/async-api-render-engine/README.md]: AsyncAPI is in dependencies array of this render-engine, odata-react in dc-odata doesn't at the moment.

File: packages/odata-render-engine/README.md [hunk 2]
```diff
@@ -20,14 +20,19 @@ The OData render engine is based on the [OData React](https://github.com/kyma-in
...
 
 To use default styles based on [SAP Fundamentals](https://sap.github.io/fundamental/), install the npm package from [this](https://sap.github.io/fundamental/getting-started.html) Getting Started guide and import the styles as follows:
 
 ```js
-import "@kyma-project/dc-odata-render-engine/lib/styles.css";
+import "@kyma-project/odata-react/lib/styles.css";
 import "fiori-fundamentals/dist/fiori-fundamentals.min.css";
 ```
+
+To use the default font `72` provided by SAP Fundamentals, import (after installation [SAP Fundamentals](https://sap.github.io/fundamental/) package) the font as follows:
```
Comment (kazydek) [inline packages/odata-render-engine/README.md]:
```suggestion
To use the default font `72` provided by SAP Fundamentals, install the [SAP Fundamentals](https://sap.github.io/fundamental/) package, and import the font as follows:
```

**Pull request merged**